pkg/mutate/squashfs.go

```diff
@@ -124,6 +141,53 @@ func (c *squashfsConverter) makeSquashfs(r io.Reader) (string, error) {
 	return path, nil
 }
 
+// convertWhiteout accepts a Layer l and returns:
+//
+//   - A ReadCloser providing the layer tar, passed through a filter that converts
+//     whiteout markers from AUFS -> OverlayFS, if c.noConvertWhiteout is false.
+//     In this case, any error from the filter will propagate via the returned channel.
+//   - A ReadCloser providing the layer tar directly, if c.noConvertWhiteout is true.
+//
+// Note that when conversion is performed, the layer is read twice.
+func (c *squashfsConverter) convertWhiteout(l v1.Layer) (io.ReadCloser, chan error, error) {
+	rc, err := l.Uncompressed()
```

Comment:
`convertWhiteout` is a little confusing here, since it may or may not convert whiteouts. Maybe use `Uncompressed` and then document the potential conversion it does?

```suggestion
// Uncompressed returns an io.ReadCloser for the uncompressed layer contents. If c.convertWhiteout
// is true, convert whiteout markers from AUFS -> OverlayFS.
func (c *squashfsConverter) Uncompressed(l v1.Layer) (io.ReadCloser, error) {
```